src/main/kotlin/no/nav/familie/ef/iverksett/arena/VedtakhendelseMapper.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.ef.iverksett.arena
+
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSApplikasjoner
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSAvslutningsstatuser
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSBehandlingstema
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSVedtakHendelser
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
Husk å legge inn familie-settings i code style :)

Reply:
Var redd jeg ikke hadde det, men ser ut til at det er lagt inn.
![Screen Shot 2021-06-09 at 09 52 32](https://user-images.githubusercontent.com/42737566/121315262-7680b580-c908-11eb-87f0-1660cc0b8edb.png)

Gjorde en liten test på den klassen, ved å sette wildcard import på de andre tingene også. Ved optimize imports i IntelliJ, så fikset den alle utenom java.util.* ?? Mulig bug i IntelliJ? Ser vi har unntak for wildcard import, men ikke for java.util. Har du lyst til å teste hos deg, for å se om optimize imports virker for denne hos deg?

Reply:
Ja jeg hadde problem med en versjon av intellij tidligere, då måtte jeg fjerne teamfamilie fra schema, og ta import på nytt.
Gjerne når jeg kun hadde 1 intellijvindue åpent... Har hatt mye krøll med intellij og scheme :P